zee/config/config.ron

```diff
@@ -248,6 +249,7 @@
                 width: 4,
                 unit: Space,
             ),
+            shebangs: ["python"],
```

Comment:
There is also another Python shebang available:  `python3`.  On some Linux systems, this difference is still important.

```suggestion
            shebangs: [
                "python",
                "python3",
            ],
```

Reply:
The regex as defined matches python/python2.7/python3/etc. so it isn't needed to define a variant for python3 in the list of shebangs.